app/templates/events/view/sessions.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="ui grid stackable">
     {{#if this.onSessionRoute}}
       <div class="row">
-        <div class="eight wide column">
+        <div class="{{if this.device.isTablet 'sixteen' 'twelve'}} wide column">
```

Comment:
Use existing classes of semantic ui. Refer to semantic ui grid.

Reply:
These are existing classes